rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1080,17 +1097,17 @@ def test_events_filter_params(rotkehlchen_api_server, ethereum_accounts, start_w
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     db = rotki.data.db
-    tx1 = make_ethereum_transaction(tx_hash=b'1', timestamp=1)
-    tx2 = make_ethereum_transaction(tx_hash=b'2', timestamp=2)
-    tx3 = make_ethereum_transaction(tx_hash=b'3', timestamp=3)
-    tx4 = make_ethereum_transaction(tx_hash=b'4', timestamp=4)
+    tx1 = make_ethereum_transaction(tx_hash=b'1', timestamp=Timestamp(1))
+    tx2 = make_ethereum_transaction(tx_hash=b'2', timestamp=Timestamp(2))
+    tx3 = make_ethereum_transaction(tx_hash=b'3', timestamp=Timestamp(3))
+    tx4 = make_ethereum_transaction(tx_hash=b'4', timestamp=Timestamp(4))
     test_contract_address = make_evm_address()
-    event1 = make_ethereum_event(tx_hash=b'1', index=1, asset=A_ETH, timestamp=TimestampMS(1), location_label=ethereum_accounts[0], product=EvmProduct.STAKING)  # noqa: E501
-    event2 = make_ethereum_event(tx_hash=b'1', index=2, asset=A_ETH, counterparty='EXAMPLE_PROTOCOL', timestamp=TimestampMS(1), location_label=ethereum_accounts[0])  # noqa: E501
-    event3 = make_ethereum_event(tx_hash=b'1', index=3, asset=A_WETH, counterparty='EXAMPLE_PROTOCOL', timestamp=TimestampMS(1), location_label=ethereum_accounts[0])  # noqa: E501
-    event4 = make_ethereum_event(tx_hash=b'2', index=4, asset=A_WETH, timestamp=TimestampMS(2), location_label=ethereum_accounts[0])  # noqa: E501
-    event5 = make_ethereum_event(tx_hash=b'4', index=5, asset=A_DAI, event_type=HistoryEventType.STAKING, event_subtype=HistoryEventSubType.DEPOSIT_ASSET, timestamp=TimestampMS(4), location_label=ethereum_accounts[2], address=test_contract_address)  # noqa: E501
-    event6 = make_ethereum_event(tx_hash=b'4', index=6, asset=A_DAI, event_type=HistoryEventType.STAKING, event_subtype=HistoryEventSubType.REMOVE_ASSET, timestamp=TimestampMS(4), location_label=ethereum_accounts[2])  # noqa: E501
+    event1 = make_ethereum_event(tx_hash=b'1', index=1, asset=A_ETH.resolve_to_crypto_asset(), timestamp=TimestampMS(1), location_label=ethereum_accounts[0], product=EvmProduct.STAKING)  # noqa: E501
```

Comment:
This resolution is not needed. The type can be more loose here for this helper function. Otherwise the tests would have failed no?

Here and in all other parts you did this change